src/pyHepGrid/src/backendManagement.py

```diff
@@ -172,6 +172,9 @@ def _do_stats_job(self, jobid_raw):
         elif "Running" in strOut:
             return self.cRUN
         elif "Failed" in strOut:
+            # if we still have a return code 0 something is odd
+            if "Exit Code: 0" in strOut:
+                return self.cMISS
```

Comment:
I think ```self.cUNK``` makes more sense here?

Reply:
No `cUNK` is used for all jobs as the default before running `arcstat` for the first time. It marks that we "do not have a status yet", as such it's not even printed. `cMISS` is not optimal but does what we want and shows that there is something that needs a manual check.

Reply:
On reflection - for these problematic jobs, do we know what the string we parse days before the successfully-running SLURM process returns?

Reply:
Good questions. I don't know if they actually change states. The example jobs don't run long enough to test that and I don't really want to submit 500 jobs that takes 15 minutes to test that.

Reply:
Probably worth monitoring, but not worth worrying about now..